src/Sylius/Bundle/UserBundle/Command/CreateUserCommand.php

```diff
@@ -0,0 +1,198 @@
+    {
+        $email = $input->getArgument('email');
+        $password = $input->getArgument('password');
+        $roles = $input->getArgument('roles');
+        $superAdmin = $input->getOption('super-admin');
+        $disabled = $input->getOption('disabled');
+
+        $securityRoles = ['ROLE_USER'];
+        if ($superAdmin) {
+            $securityRoles[] = 'ROLE_SYLIUS_ADMIN';
+            $securityRoles[] = 'ROLE_ADMINISTRATION_ACCESS';
+        }
+
+        $user = $this->createUser(
+            $email,
+            $password,
+            !$disabled,
+            $roles,
+            $securityRoles
+        );
+
+        $this->getEntityManager()->persist($user);
+        $this->getEntityManager()->flush();
+
+        $output->writeln(sprintf('Created user <comment>%s</comment>', $email));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        if (!$input->getArgument('email')) {
+            $email = $this->getHelper('dialog')->askAndValidate(
+                $output,
+                'Please enter an email:',
+                function($username) {
+                    if (empty($username)) {
+                        throw new \Exception('Email can not be empty');
+                    }
+                    return $username;
+                }
+            );
+
+            $input->setArgument('email', $email);
+        }
+
+        if (!$input->getArgument('password')) {
+            $password = $this->getHelper('dialog')->askHiddenResponseAndValidate(
+                $output,
+                'Please choose a password:',
+                function($password) {
+                    if (empty($password)) {
+                        throw new \Exception('Password can not be empty');
+                    }
+                    return $password;
+                }
+            );
+
+            $input->setArgument('password', $password);
+        }
+
+        if (!$input->getArgument('roles')) {
+            $roles = $this->getHelper('dialog')->ask(
+                $output,
+                'Please enter user\'s roles (separated by space):'
+            );
+
+            if (!empty($roles)) {
+                $input->setArgument('roles', explode(' ', $roles));
+            }
+        }
+    }
+
+    /**
+     * @param string $email
+     * @param string $password
+     * @param bool $enabled
+     * @param array $roles
+     * @param array $securityRoles
+     *
+     * @return UserInterface
+     */
+    protected function createUser($email, $password, $enabled, array $roles, array $securityRoles = array('ROLE_USER'))
+    {
+        $canonicalizer = $this->getContainer()->get('sylius.user.canonicalizer');
+
+        /**
+         * @var $user UserInterface
+         * @var $customer CustomerInterface
+         */
+        $user = $this->getUserRepository()->createNew();
+        $user->setUsername($email);
+        $user->setEmail($email);
```

Comment:
The `User` can't exist without `Customer` and the `email` field doesn't exist on user entity https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/User/Model/User.php#L414.

Take a look how is it handled in the Behat context https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Behat/CoreContext.php#L717 and repeat the logic here.